main.go

```diff
@@ -56,6 +57,8 @@ var (
 
 var defaultConfFile string = "/etc/mender/mender.conf"
 
+var DeploymentLogger *DeploymentLogManager
```

Comment:
Matter of taste I think. It might be easier with pointer to check if it is nil or not. 
What is more, in our log library `Log` is also a pointer.